internal/install/install.go

```diff
@@ -44,7 +44,7 @@ func EnsureInstalled() error {
 		return errors.Wrap(err, "writing static resources failed")
 	}
 
-	fmt.Println("elastic-package has been installed.")
+	fmt.Fprintln(os.Stderr, "elastic-package has been installed.")
```

Comment:
Why the change to `os.Stderr`? Also, should this use the logger?

Reply:
No, in this case, the output is evalled from Stdout:

`eval "$(elastic-package stack shellinit)"`

 so this `fmt.Println` goes to Stderr. Another option would be replace it with `# comment`

Reply:
One more note on how did I find it. The eval failed only during the first run of a new build, whenever docker images have been updated (reinstallation notice).

Reply:
Hmmm, I see. Feels a bit hacky, especially since all other output goes to STDOUT but I don't see a better option than what you've done (writing to STDERR) without complicating the code. I'm good with this change.